style.css

```diff
@@ -1,8 +1,104 @@
 body {
   margin: 0;
-  background: tomato;
+  background: white;
+  font-family: sans-serif;
 }
 
- * {
-   box-sizing: border-box;
- }
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+img {
+  max-width: 100%;
+  border-radius: 20rem;
+  position: absolute;
+  left: 0;
+  top: 16%;
```

Comment:
Don't use a guessing unit here and use classes instead styling all images so that you can have other images on the page that don't get overstyled.